maestro/server/pipeline.yaml

```diff
@@ -0,0 +1,29 @@
+serviceGroup: Microsoft.Azure.ARO.Test
+rolloutName: Maestro Server Rollout
+steps:
+- name: deploy
+  subscription: {{ .serviceClusterSubscription }}
+  resourceGroup: {{ .serviceClusterRG }}
+  action:
+    type: Shell
+    command: "make deploy"
+    # we could infer the pwd from the location of this file and could avoid make -C
+    variables:
+    - name: EVENTGRID_NAME
+      value: maestroEventgridName
```

Comment:
What does this mean? I think you need the `{{ .maestro.eventGridName }}` syntax here?

Reply:
as discussed, let's keep this a name based ref and refactor later to event not require this section at all.
for the time being i renamed `value` to `configRef` to make it clearer this is a ref to a config and not a value